src/Redux/Reducers/AuthSlice.tsx

```diff
@@ -0,0 +1,61 @@
+import {createSlice,createAsyncThunk} from "@reduxjs/toolkit";
+import {
+  createUserWithEmailAndPassword,
+  signInWithEmailAndPassword,
+} from "firebase/auth";
+import {auth} from "./../../fireBase"
+import {InitialState,Details} from  "./Types";
+
+// commented code on this page is not removed for further use
+export const logIn = createAsyncThunk("auth/logIn", async function (details:Details){
+  const email = details.email,
+  password = details.password;
+  try {
+    const response = await signInWithEmailAndPassword(auth, email, password);
+    // localStorage.setItem("user", JSON.stringify(user ? user : null));
+   console.log(response);
+    // return user;
```

Comment:
not sure! maybe you can extract and return only "uid" and "name" as an object from response, which will be passed as payload to fulfilled case reducer

Reply:
Yeah I agree to you Vishnu that's why on top I wrote about the commented code because I want to use that code as of now what would I need not sure that's why left it commented
Thanks for reviewing though